schemas/xml/AAS.xsd

```diff
@@ -118,9 +118,9 @@
 			<element maxOccurs="unbounded" minOccurs="0" name="isCaseOf" type="aas:reference_t"/>
 		</sequence>
 	</complexType>
-	<complexType name="conceptDescriptionRefs_t">
+	<complexType name="conceptDescription_t">

```

Comment:
conceptDescription_t would be a completely different type than conceptDescriptionRefs. Probably both are needed or one can be removed completly.